clients/fides-js/src/fides-ext-gpp.ts

```diff
@@ -11,6 +11,7 @@ import {
   CmpDisplayStatus,
```

Comment:
This pattern of relying on Fides events to do GPP updates worked quite well, and I think we should consider porting the TCF code to do the same (right now in the TCF code, we manually call the TCF updates to generate the string, then the listeners just set the string. instead, I think it may make sense to let the listeners do all the string generation too, to keep it one file, and have the components not need to worry about what functions they are importing). this would help separate our bundles a bit more!